bedrock/firefox/templates/firefox/whatsnew/includes/fx96-en/s2d.html

```diff
@@ -0,0 +1,23 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+<div class="mzp-c-emphasis-box">
+  <h3>Get Firefox for mobile</h3>
+{% if show_send_to_device %}
+  <p>Email yourself a download link</p>
+  {{ send_to_device(include_title=False, message_set='fx-whatsnew-96', spinner_color='#ccc;', class='vertical', id='send-to-device') }}
+{% else %}
+  <p>Scan the QR code to get started</p>
+  <div class="qr-code-wrapper">
+    <img src="{{ static('img/firefox/mobile/protocol/qr-firefox.png') }}"
+        id="firefox-qr"
+        data-mozillaonline-link="{{ static('img/firefox/mobile/protocol/qr-firefox-mozillaonline.png') }}"
+        alt="Scan the QR code to get Firefox on mobile">
+  </div>
+{% endif %}
```

Comment:
Probably so, and I would guess it's just because en-CA was only somewhat recently activated and the widget predates it.

It would be easy enough to add it to the list in settings, but that might impact every other instance of the widget so maybe best to do it separately rather than in this PR.